### DISCUSSION TITLE
Add timezone support for interactive embedded grafana graphs

### DISCUSSION THREAD

**Discussion by khems** [grafana/grafana#40061] [Dashboards, Panels & Visualizations]
https://grafana.com/docs/grafana/latest/sharing/share-panel/

For server rendered images, graphs can be requested with given **time zones**.
"tz: timezone in the format UTC%2BHH%3AMM where HH and MM are offset in hours and minutes after UTC"

I think it would be very useful if this could also be **added to the query string parameters for embedding a panel**.

My specific use case is a for embedding on a website which allows users to pick a timezone rather than relying on browser time. What I'm trying to avoid is making users also have to change the timezone in the Grafana preferences panel. 

If there's another way around this like if there's an endpoint for setting the users time zone that I could hit when the user changes the timezone on the website then please let me know.

**Comment by khems** [comment]
A way around this for anyone with the same issue is to send a PUT request to /grafana/api/user/preferences with the required timezone.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
